mybot.py

```diff
@@ -211,6 +211,12 @@ def get_game_stats(message, year, week, team):
 	game = nflgame.one(year,week=week,home=team.upper(),away=team.upper())
 	
 	# TODO: build actual validation for the year, week, team and improve the error reporting
+	if len(year) <> 4:
+		message.reply("NFL has been around for only the past 50ish years. Might want to try a 4 digit year")
+		return
+	if week > 25:
```

Comment:
I think nflgame only understands weeks 1-17 and has a separate designation for playoff games by using the 'kind' parameter. For now let's keep this to regular season? Or, if you're feeling ambitious build in the kind validation? 
http://pdoc.burntsushi.net/nflgame